core/src/views/element.rs

```diff
@@ -2,17 +2,91 @@ use crate::{Context, Handle, View};
 
 /// A basic element with no interactivity.
 ///
+/// An element has no children and no special rendering logic. It can be used to render a rectangle,
+/// a line or a circle using view styling rules.
```

Comment:
Another useful thing you can do with an element is add a image backgound or a gradient.